src/frameworks/drupal8/solr.md

```diff
@@ -1,7 +1,30 @@
 # Using Solr with Drupal 8.x
 
-## Requirements
+## Steps
+  - Enable Solr Service
+  - Enable the Solr relationship
+  - Download and enable the modules
+  - Configure Search API Solr
+
+### Enable Solr Service
+Different Solr versions are supported. Advanced configuration may be found at the [Solr Service documentation page](../../configuration/services/solr.md).
+
+Basically, uncomment or add these lines to your ``.platform/services.yaml``:
```

Comment:
"Basically" isn't good verbage for documentation.  Let's follow the structure and language pattern from the Redis page for consistency.